src/transformers/models/informer/modeling_informer.py

```diff
@@ -898,7 +898,7 @@ def forward(
         L_Q = query_states.size(1)
         log_L_Q = np.ceil(np.log1p(L_Q)).astype("int").item()
 
-        U_part = min(self.factor * log_L_K, L_K)
+        U_part = min(self.factor * L_Q * log_L_K, L_K)
```

Comment:
@kashif please note this fix

Source: https://github.com/zhouhaoyi/Informer2020/issues/161